salt/minion.py

```diff
@@ -2482,6 +2482,7 @@ def pillar_refresh(self, force_refresh=False):
                 self.opts["id"],
                 self.opts["saltenv"],
                 pillarenv=self.opts.get("pillarenv"),
+                force_refresh=True,
```

Comment:
`force_refresh` is passed through elsewhere, should it really be hardcoded here rather than passing it through?

Reply:
This is a good point, I will make it configurable, so the user can choose not to refresh the pillar cache. I also think I am going to rename `force_refresh`, since this is a kwarg already used in `saltutil.refresh_pillar` used for refreshing modules.